lib/molecule-dev-environment.js

```diff
@@ -98,6 +300,45 @@ export default {
             ),
           );
         },
+        "molecule:free-terminal-toggle": () => {
```

Comment:
All these commands can be removed. Settings can and should be changed directly from the settings view in Atom.